app/switchboard/Switches.scala

```diff
@@ -14,13 +17,34 @@ object Switches {
     Switches(
       PaymentMethodsSwitch.fromConfig(config.getConfig("oneOff")),
       PaymentMethodsSwitch.fromConfig(config.getConfig("recurring")),
+      experimentsFromConfig(config, "abtests"),
       SwitchState.fromConfig(config, "optimize"),
       SwitchState.fromConfig(config, "internationalSubscribePages")
     )
 
+  def experimentsFromConfig(config: Config, rootKey: String): Map[String, ExperimentSwitch] =
+    config.getConfigList(rootKey).asScala.map(ExperimentSwitch.fromConfig).map { x => x.name -> x }.toMap
```

Comment:
`x` would be `switch`?

Reply:
no, it would be `x :: { name :: String | _ }` using row-polymorphic notation 😃